src/lib.rs

```diff
@@ -74,6 +74,10 @@ pub mod base;
 #[macro_use]
 pub mod system;
 
+// Import all top-level HII definitions and structures. These don't readily fall under a protocol namespace,
```

Comment:
@dvdhrm , wasn't quite sure how best to bring in the root-level module. Let me know if this (or the comment) needs to change. Thanks!